Makefile

```diff
@@ -52,11 +52,11 @@ all:
 	@echo "Current type detected is: $(type)"
 
 recap.cron:
-	@sed -e 's|@BINDIR@|$(BINDIR)|' src/utils/recap.cron.in > src/utils/recap.cron
+	@sed -e 's|\(^BINDIR=\).*|\1$(BINDIR)|' src/utils/recap.cron.in > src/utils/recap.cron
 
 recap.systemd:
 	@for service_file in src/utils/*.service.in; do \
-	sed -e 's|@BINDIR@|$(BINDIR)|' $${service_file} > $${service_file%.in}; \
+	sed -e 's|\(=BINDIR=\).*|\1$(BINDIR)|' $${service_file} > $${service_file%.in}; \
```

Comment:
Can you add the full regex for the line, i.e. include terminators `^$`.

Reply:
Fixed slightly differently. Please check out the commit and let me know.